kolibri/content/management/commands/importchannel.py

```diff
@@ -119,6 +123,13 @@ def _transfer(self, method, channel_id, baseurl=None, path=None):
                     except IOError:
                         pass
```

Comment:
Could you add a comment or log entry here, while we're at it? :)

> `"Tried to remove {}, but exception {} occurred".format(dest, e)`

What I mean is that the file isn't expected to be missing, so if it is.. eh that's weird.

Reply:
Yes I will add a log here